src/docker.ts

```diff
@@ -127,6 +127,10 @@ async function getSHATagWithPrefix(repository: string): Promise<string> {
   } else if (ref !== '' && ref.startsWith('refs/tags')) {
     // on push to tags
     shaWithPrefix = `tag-${sha}`;
+  } else if (ref.startsWith('refs/heads/release')) {
+    // on push to any release branch
+    const branchName = ref.replace(/^refs\/heads\//, '');
```

Comment:
the purpose of this code is to have a fixed set of image prefixes in order to customize the lifecycle policy in ECR, that prefix is more like a type of branch and not the name of the branch, please just use `release-${sha}`